unit/constant/generate_constants.go

```diff
@@ -0,0 +1,334 @@
+)
+
+var constants = []Constant{
+	{
+		Name: "AtomicMass", Value: 1.660539040e-27,
+		Dimensions:  []Dimension{{massName, 1}},
+		Comment:     "AtomicMass is the atomic mass constant (mᵤ), one twelfth of the mass of an unbound atom of carbon-12 at rest and in its ground state.",
+		Uncertainty: 0.000000020e-27,
+	},
+	{
+		Name: "Avogadro", Value: 6.022140857e23,
+		Dimensions:  []Dimension{{moleName, -1}},
+		Comment:     "Avogadro is the Avogadro constant (A), the number of constituent particles contained in one mole of a substance.",
+		Uncertainty: 0.000000074e23,
+	},
+	{
+		Name: "Boltzmann", Value: 1.38064852e-23,
+		Dimensions:  []Dimension{{massName, 1}, {lengthName, 2}, {timeName, -2}, {temperatureName, -1}},
+		Comment:     "Boltzmann is the Boltzmann constant (k), it relates the average relative kinetic energy of particles in a gas with the temperature of the gas.",
+		Uncertainty: 0.00000079e-23,
+	},
+	{
+		Name: "ElectricConstant", Value: 1 / (4 * math.Pi * 1e-7 * lightSpeed * lightSpeed),
+		Dimensions: []Dimension{{currentName, 2}, {timeName, 4}, {massName, -1}, {lengthName, -3}},
+		Comment:    "ElectricConstant is the electric constant (ε₀), the value of the absolute dielectric permittivity of classical vacuum.",
+	},
+	{
+		Name: "ElementaryCharge", Value: elementaryCharge,
+		Dimensions:  []Dimension{{currentName, 1}, {timeName, 1}},
+		Comment:     "ElementaryCharge, is the elementary charge constant (e), the magnitude of electric charge carried by a single proton or electron.",
+		Uncertainty: 0.0000000098e-19,
+	},
+	{
+		Name: "Faraday", Value: 96485.33289,
+		Dimensions:  []Dimension{{currentName, 1}, {timeName, 1}, {moleName, -1}},
+		Comment:     "Faraday is the Faraday constant, the magnitude of electric charge per mole of electrons.",
+		Uncertainty: 0.00059,
+	},
+	{
+		Name: "FineStructure", Value: fineStructure,
+		Comment:     "FineStructure is the fine structure constant (α), it describes the strength of the electromagnetic interaction between elementary charged particles.",
+		Uncertainty: 0.0000000017e-3,
+	},
+	{
+		Name: "Gravitational", Value: 6.67408e-11,
+		Dimensions:  []Dimension{{massName, -1}, {lengthName, 3}, {timeName, -2}},
+		Comment:     "Gravitational is the universal gravitational constant (G), the proportionality constant connecting the gravitational force between two bodies.",
+		Uncertainty: 0.00031e-11,
+	},
+	{
+		Name: "LightSpeedInVacuum", Value: lightSpeed,
+		Dimensions: []Dimension{{lengthName, 1}, {timeName, -1}},
+		Comment:    "LightSpeedInVacuum is the c constant, the speed of light in a vacuum.",
+	},
+	{
+		Name: "MagneticConstant", Value: 2 * fineStructure * planck / (elementaryCharge * elementaryCharge * lightSpeed),
+		Dimensions: []Dimension{{currentName, 2}, {timeName, 4}, {massName, -1}, {lengthName, -3}},
+		Comment:    "MagneticConstant is the magnetic constant (μ₀), the magnetic permeability in a classical vacuum.",
+	},
+	{
+		Name: "Planck", Value: planck,
```

Comment:
We should also have ReducedPlank (hbar). Doesn't have to happen this PR.

Reply:
Can happen in the update.